charts/jx-app-jacoco/templates/deployment.yaml

```diff
@@ -42,7 +42,9 @@ spec:
 {{ toYaml .Values.resources | indent 12 }}
         env:
         - name: TEAM_NAMESPACE
-          value: {{ default "jx" .Values.teamNamespace}}
+          valueFrom:
```

Comment:
@jglick, I think this gives us what we want. This will inject the install namespace into the pod as an environment variable. This means the app is monitoring the same namespace it got installed under via `jx add app -n <namespace> foo`.

Reply:
Yeah you can do it that way. IIUC this amounts to the same thing as the pod loading its own namespace at runtime; either way, you are not relying on Helm tricks since this is baked into the `Deployment`.

Reply:
> Yeah you can do it that way. IIUC this amounts to the same thing as the pod loading its own namespace at runtime; either way, you are not relying on Helm tricks since this is baked into the Deployment.

Right. I also like it is not a Helm trick or similar. It uses basic Kubernetes functionality while not having to break the ability to run locally.